cnf-tests/testsuites/pkg/pods/pods.go

```diff
@@ -76,6 +76,14 @@ func RedefinePodWithNetwork(pod *corev1.Pod, networksSpec string) *corev1.Pod {
 	return pod
 }
 
+// RedefineAsPrivileged updates the pod definition to be privileged
+func RedefineAsPrivileged(pod *corev1.Pod) *corev1.Pod {
+	pod.Spec.Containers[0].SecurityContext = &corev1.SecurityContext{}
```

Comment:
this will override the securityContext please check before that if it's already there just add the true